README/ReleaseNotes/v620/index.md

```diff
@@ -441,6 +445,10 @@ shuts down the interpreter.
 `ctypes` types (`c_long`, `c_int`, `c_double`).
 - Added TPython.
 - Added support for `from ROOT import *` in Python2 (only).
+- python-dev is now required and the compilation will fail if CMake is not able
```

Comment:
I think the comment of @oshadura was not to repeat exactly in the Python section what you wrote in the Build section. @oshadura , can you tell @maxgalli exactly what you would like to have here?

Reply:
Let's ignore this part then, I am not sure what to say here.